assets/static/js/dataset-submission.js

```diff
@@ -33,6 +33,47 @@ $(function() {
 
     $("html").show();
 
+    const selectedFunders = [];
+
+    function removeItemFromList(arr, value) {
+        var index = arr.indexOf(value);
+        if (index > -1) {
+          arr.splice(index, 1);
+        }
+        return arr;
+    }
+
+    const addedFunders = [];
+
+    $.ajax({
+        url: Routing.generate('pelagos_api_datasets_get_collection') + '?udi=' + $("#regForm").attr("udi") + '&_properties=funders.id',
+        success: function(data){
+            addedFunders.push
+              let associatedFunders = data[0].funders;
```

Comment:
this reference to data[0] in console shows "Uncaught TypeError: data[0] is undefined"

Reply:
I should've added a check, good catch

Reply:
Fixed it